go.mod

```diff
@@ -407,7 +410,7 @@ replace xorm.io/xorm => github.com/grafana/xorm v0.8.3-0.20220614223926-2fcda756
 
 // Use our fork of the upstream alertmanagers.
 // This is required in order to get notification delivery errors from the receivers API.
-replace github.com/prometheus/alertmanager => github.com/grafana/prometheus-alertmanager v0.24.1-0.20221012142027-823cd9150293
+replace github.com/prometheus/alertmanager v0.25.0 => github.com/grafana/prometheus-alertmanager v0.24.1-0.20221223152405-14ee77b330f2
```

Comment:
Why is this still here? Do we need a version of the prometheus alertmanager that's newer than the v0.25.0 release? In which case I think you could just require the exact commit that we need by doing a `go get github.com/prometheus/alertmanager@$COMMIT_HASH`

Reply:
Note that we're not replacing Prometheus' AM with a newer version of the same repo, but with our custom fork of the AM. This fork contains changes that we're hoping to upstream eventually, including all the work done in the [Errors in notification pipeline epic](https://github.com/grafana/alerting-squad/issues/289).

Reply:
`/prometheus/alertmanager != /grafana/prometheus-alertmanager`

Reply:
We need to do a new release of https://github.com/grafana/prometheus-alertmanager though, otherwise `grafana/prometheus-alertmanager` 0.24 is still that latest version, now based on `prometheus/alertmanager` 0.25, which looks strange.